src/main/java/org/galatea/starter/service/IexService.java

```diff
@@ -45,5 +73,50 @@
     }
   }
 
+  /**
+   * Get the last traded price for each Symbol that is passed in.
+   *
+   * @param symbol the symbol to get a historical prices for.
+   * @param range the range to get historical prices for
+   * @param date the date to get historical prices for
+   * @return a list of historical prices for the Symbol that is passed in.
+   */
+  public List<IexHistoricalPrices> getHistoricalPricesForSymbol(
```

Comment:
This is a very long method. Please split it into methods that are ideally <10 lines